[2024-04-12T07:19:21.163+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-12T07:19:21.261+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_complex.get_entry_group manual__2024-04-12T07:19:14.791350+00:00 [queued]>
[2024-04-12T07:19:21.276+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_complex.get_entry_group manual__2024-04-12T07:19:14.791350+00:00 [queued]>
[2024-04-12T07:19:21.278+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-12T07:19:21.341+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): get_entry_group> on 2024-04-12 07:19:14.791350+00:00
[2024-04-12T07:19:21.388+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3837) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-12T07:19:21.393+0000] {standard_task_runner.py:63} INFO - Started process 3846 to run task
[2024-04-12T07:19:21.385+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'get_entry_group', 'manual__2024-04-12T07:19:14.791350+00:00', '--job-id', '7', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmp26oa01p0']
[2024-04-12T07:19:21.412+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask get_entry_group
[2024-04-12T07:19:21.510+0000] {task_command.py:426} INFO - Running <TaskInstance: example_complex.get_entry_group manual__2024-04-12T07:19:14.791350+00:00 [running]> on host 48c2eff06d44
[2024-04-12T07:19:21.790+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_complex' AIRFLOW_CTX_TASK_ID='get_entry_group' AIRFLOW_CTX_EXECUTION_DATE='2024-04-12T07:19:14.791350+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-12T07:19:14.791350+00:00'
[2024-04-12T07:19:21.795+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-12T07:19:21.796+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-12T07:19:21.800+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo get_entry_group']
[2024-04-12T07:19:21.831+0000] {subprocess.py:86} INFO - Output:
[2024-04-12T07:19:21.836+0000] {subprocess.py:93} INFO - get_entry_group
[2024-04-12T07:19:21.836+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-12T07:19:21.837+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-12T07:19:21.890+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=get_entry_group, execution_date=20240412T071914, start_date=20240412T071921, end_date=20240412T071921
[2024-04-12T07:19:21.939+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-12T07:19:21.967+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-12T07:19:21.973+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
